homeassistant/components/sonos/speaker.py

```diff
@@ -485,19 +485,23 @@ def async_update_volume(self, event: SonosEvent) -> None:
     #
     # Speaker availability methods
     #
+    @callback
+    def _async_reset_seen_timer(self):
+        """Reset the _seen_timer scheduler."""
+        if self._seen_timer:
+            self._seen_timer()
+        self._seen_timer = self.hass.helpers.event.async_call_later(
```

Comment:
Side note: Nowadays we prefer importing the event helpers to have working type annotations.